build.gradle

```diff
@@ -256,8 +256,8 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: versions.springBoot
 
     compile group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '2.5.6'
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.2'
-    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.2'
+    //compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.2'
+    //compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.2'
```

Comment:
Remove commented out lines, as I see you've moved them below (line 279)

Reply:
ok